circe/src/test/scala/org/http4s/circe/CirceSensitiveDataEntityDecoderSpec.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2015 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.circe
+
+import cats.data.EitherT
+import cats.effect.IO
+import io.circe.{Decoder, HCursor, Json}
+import io.circe.syntax._
+import org.http4s.{DecodeFailure, Http4sSpec, InvalidMessageBodyFailure, Response, Status}
+
+object CirceSensitiveDataEntityDecoderSpec {
+
+  private final case class Person(ssn: String)
+  private object Person {
+    implicit val decoder: Decoder[Person] = new Decoder[Person] {
+      override def apply(c: HCursor): Decoder.Result[Person] =
+        c.downField("ssn").as[String].map(Person(_))
+    }
+  }
+
+}
+
+class CirceSensitiveDataEntityDecoderSpec extends Http4sSpec {
```

Comment:
Can we define new specs with MUnit instead?  We've started a migration, and it would be nice if the new ones started in the new world.  It will also save you the `unsafeRunSync()`s.

Reply:
Sure!

Reply:
Can I blame Daniel Spiewak for my usage of specs2 here re: https://www.youtube.com/watch?v=jWJ5A1irH_E&list=PLSUh6oJ5ZotUS0k5d8EcHJIN9ATVJ_SfL? 😀

Reply:
Oh, I've always liked specs2 as well.  Still do.  MUnit is just a bit more straightforward for truly asynchronous testing, and the literate DSLs have fallen out of fashion.

Reply:
Please see https://github.com/http4s/http4s/pull/4012/commits/beb9ee5f48fa8c20ead7f75555a57171112d9382, Ross.